src/rpc/net.cpp

```diff
@@ -186,6 +186,8 @@ static RPCHelpMan getpeerinfo()
                                                               "best capture connection behaviors."},
                     {RPCResult::Type::STR, "transport_protocol_type", "Type of transport protocol: \n" + Join(TRANSPORT_TYPE_DOC, ",\n") + ".\n"},
                     {RPCResult::Type::STR, "session_id", "The session ID for this connection, or \"\" if there is none (\"v2\" transport protocol only).\n"},
+                    {RPCResult::Type::NUM, "misbehavior_score", "The accumulated misbehavior score for this peer.\n"
+                                                                 "The peer will be disconnected if 100 is reached.\n"},
```

Comment:
Disconnected or banned?

Reply:
Disconnected and discouraged, no?

Reply:
Technically yes, but it feels like the disencoragement is an implementation detail completely transparent to the end user. I feel it's not worth mentioning, but I won't oppose it if there's consensus

Reply:
IMO it's ok to leave as is.

Reply:
Based on my reading of `banman.h`, we could add something like: "Their IP is not banned, but subsequent connections from the same address are given lower priority (see banman.h)"

"lower priority" is a bit vague, but I think it more clearly captures what "discourage" is trying to do: priotize _other_ peers, but tolerate them in case they give us something useful.

Reply:
If we are going to refer them to `banman.h`, I think just adding `and discouraged` should be enough. Saying the peer is "discouraged" or that "connections to it are given lower priority" is pretty vague without context, so I'd rather keep the help short if add a reference